lib/Minz/Log.php

```diff
@@ -98,6 +98,7 @@ protected static function ensureMaxLogSize($file_name) {
 				rewind($fp);
 				ftruncate($fp, 0);
 				fwrite($fp, $content ? $content : '');
+				fwrite($fp, '[' . date('r') . "] [notice] --- Log rotate.\n";);
```

Comment:
I am not a big fan of string concatenation. I'd like ```sprintf``` better. My 2¢

Reply:
Is PHP one of the languages where it makes a difference in speed? In this case it'd make no difference but if you end up with `%s%s%s` then concat can still be clearer.

(Ideally I suppose you'd have something more like Python's string.format instead.)

Reply:
Updated to `sprintf()` for @aledeg :-)

Reply:
@Frenzie @Alkarex it makes a difference on how the memory is allocated. String concatenation use more memory than sprintf.

Reply:
@aledeg In that case I too prefer sprintf, although without seeing benchmarks it might be premature optimization. ;-D

Reply:
@Frenzie I know that from one of the PHP core dev. He made a speak about optimization and that was one of it.